### ISSUE TITLE
[Android Only] Have problem with fragmentTag.

### ISSUE THREAD

**Issue by yestay90** [PSPDFKit/react-native#352]
# Details
The usage in project:
```
<PSPDFKitView
              document={localFile}
              configuration={Platform.OS === 'ios' ? iOSOptions : androidOptions}
              fragmentTag="pspdfFragmentTag"
              ref={ref => {
                this.pspdfInstance = ref;
              }}
              style={{ flex: 1, color: '#267AD4', backgroundColor: 'red' }}
              onDocumentSaved={this.documentSaved}
              onAnnotationsChanged={this.annotationEdited}
              enableAnnotationEditing
              onDocumentSaveFailed={e => {
                Alert.alert(`Document couldn't be saved: ${e.error}`);
              }}
              onStateChanged={this.pspdfStateChange}
              annotationAuthorName={userName}
            />
```
The error: 
Error description
2020-02-26 13:00:22.340 5776-5776/io.companyName E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.companyName, PID: 5776
    java.lang.IllegalArgumentException: No view found for id 0x7f0801e8 (io.companyName/pspdf__activity_fragment_container) for fragment PdfFragment{27bc125 (f28aabef-b43d-4c2b-afc2-85ab645ac81d) id=0x7f0801e8 PSPDFKit.Fragment}

System:
    OS: macOS Mojave 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz
    Memory: 958.76 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.15.3 - /usr/local/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 26, 27, 28, 29
      Build Tools: 27.0.3, 28.0.3, 29.0.1
      System Images: android-24 | Google APIs Intel x86 Atom, android-24 | Google Play Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5977832
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5

# Acceptance Criteria

**Comment by Nlammertyn** [comment]
Can confiirm this was introduced in 1.27.0.

**Comment by irgendeinich** [comment]
Thanks for your patience. 

I tried putting the view as described in the issue into our catalog app but I wasn't able to reproduce the crash. Could you maybe try reproducing this in our catalog app or provide a small sample app that illustrates this crash, that way I can try to fix it in our wrapper.

All the best
Reinhard

**Comment by wbercx** [comment]
I consistently have this issue on React Native 0.61.5.

In development mode:

1. Open a PDF.
2. Refresh the app. (CMD + R on iOS / R+R on Android)
3. Open the PDF again.
4. Crash.

<details>
<summary>Stack trace</summary>
<p>

````
FATAL EXCEPTION: main
03-16 12:56:38.269  6292  6292 E AndroidRuntime: Process: com.my_app, PID: 6292
03-16 12:56:38.269  6292  6292 E AndroidRuntime: java.lang.IllegalArgumentException: No view found for id 0x7f0800fb (com.my_app:id/pspdf__activity_fragment_container) for fragment PdfFragment{678cedd (9de7fb2d-8c03-42fb-87bb-7a07307ad98b) id=0x7f0800fb PSPDFKit.Fragment}
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:875)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:439)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManagerImpl.java:2079)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1869)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1824)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1727)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:883)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7319)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
03-16 12:56:38.269  6292  6292 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:934)
````

</p>
</details>

**Comment by irgendeinich** [comment]
@wbercx 
Thanks for the information. Sadly in our catalog app even after updating to react-native `0.61.5` I still wasn't able to reproduce this. 

Do you have any more information you can provide or maybe a small sample app?

**Comment by irgendeinich** [comment]
I have an update on this issue. I managed to reproduce this and it will need a new framework release to fix this. I will update this ticket as soon as the fix is part of the wrapper.

**Comment by wbercx** [comment]
@irgendeinich I suspect you've already caught this as per what you wrote above, but it consistently crashes if when try to open a PDF from a button in a [modal](https://github.com/react-native-community/react-native-modal).

I don't know the correct terminology, but these modals seem somewhat disconnected from the main app. It's a known issue that if you refresh the app with a modal open, that modal will become a permanent zombie until you kill and start the app from scratch.

Perhaps that disconnect causes PDFKit to crash.

**Comment by irgendeinich** [comment]
@wbercx 
Thanks for the additional information. I just released 1.29.0 which contains the fix for this issue. Can you let me know if you are still able to reproduce this after updating?

**Comment by wbercx** [comment]
@irgendeinich Seems to work. Thanks!
